src/pymor/reductors/neural_network.py

```diff
@@ -126,7 +127,7 @@ def reduce(self, hidden_layers='[(N+P)*3, (N+P)*3]', activation_function=torch.t
             # input and output size of the neural network are prescribed by the
             # dimension of the parameter space and the reduced basis size
             assert isinstance(hidden_layers, list)
-            layers = self._compute_layers_sizes(hidden_layers)
+            layers_sizes = self._compute_layers_sizes(hidden_layers)
```

Comment:
I have a very nitpicky comment: in compound expressions like this only the last word should be pluralized.

Reply:
Ah, that indeed reads much better. I will fix this. Thanks!